packages/concatjs/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "declaration": true,
+        "module": "umd",
+        "moduleResolution": "node",
+        "types": ["node"],
+        "target": "es5",
+        "downlevelIteration": true,
+        "rootDirs": [
+            ".",
+            "../../bazel-out/darwin-fastbuild/bin/packages/concatjs",
```

Comment:
CI error is that you're missing some possible paths here ```ERROR: /home/circleci/rules_nodejs/packages/concatjs/BUILD.bazel:23:11: Compiling TypeScript project //packages/concatjs:bazel_concatjs_lib [tsc -p packages/concatjs/tsconfig.json] failed: (Exit 2): tsc.sh failed: error executing command bazel-out/k8-opt-exec-2B5CBBC6/bin/external/npm/typescript/bin/tsc.sh --project packages/concatjs/tsconfig.json --outDir bazel-out/k8-opt-exec-2B5CBBC6/bin/packages/concatjs --rootDir packages/concatjs ... (remaining 3 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox
packages/concatjs/index.ts(131,15): error TS2307: Cannot find module './internal/tsc_wrapped' or its corresponding type declarations.```

Reply:
```
k8-opt-exec-2B5CBBC6
darwin-opt-exec-2B5CBBC6
windows-opt-exec-2B5CBBC6
```
These are the outputs trees for the host (2B5CBBC6) execution platform

Reply:
I don't recall ever seeing a `linux-*` platform. Where did those ones come from?